app/conf/Configuration.scala

```diff
@@ -229,6 +229,8 @@ class ApplicationConfiguration(val playConfiguration: PlayConfiguration, val isP
   object faciatool {
     lazy val breakingNewsFront = "breaking-news"
     lazy val frontPressToolQueue = getString("frontpress.sqs.tool_queue_url")
+    lazy val issuePublishedEventsQueue = getString("publish_events.queue_url")
+      .getOrElse(throw new BadConfigurationException(s"property of type string not configured for `publish_events.queue_url` for stage $stageFromProperties"))
```

Comment:
minor: we can reuse [`getMandatoryString`](https://github.com/guardian/facia-tool/blob/master/app/conf/Configuration.scala#L41-L42) here

Reply:
good spot, thanks